### DISCUSSION TITLE
GeoCouch installation for CouchDB fails

### DISCUSSION THREAD

**Discussion by sudhir-naico** [apache/couchdb#3655] [General] [answered]
We have Apache CouchDB 2.1.1 in Ubuntu 16.04. We have installed the CouchDB by compiling from the source. We tried to install geoCouch in the server as described in the site "https://github.com/couchbase/geocouch".

Build instruction says:-
./bootstrap
./configure
make dev

But I could not find any bootstrap file in couchDb source folder.

We skipped these lines assuming we have already compiled from the source and we can just follow the other steps.

But "COUCH_SRC=<path-to-couchdb-source>/src/couchdb make couchdb" step also giving us following errors:- 
"
**Compiling /home/geocouch/vtree/src/vtree_split.erl failed:
/home/geocouch/vtree/src/vtree_split.erl:22: can't find include file "couch_db.hrl"
/home/geocouch/vtree/src/vtree_split.erl:286: undefined macro 'LOG_ERROR/2'
/home/geocouch/vtree/src/vtree_split.erl:273: function create_split_candidates/5 undefined
/home/geocouch/vtree/src/vtree_split.erl:275: spec for undefined function create_split_candidates/5
ERROR: compile failed while processing /home/geocouch/vtree: rebar_abort
Makefile:28: recipe for target 'compile' failed
make: *** [compile] Error 1**
"

We tried CouchDB 2.3.1 on Ubuntu 18.04 also, but that too doesn't help us.

Can anyone help us?

**Comment by nickva** [comment, marked as answer]
@sudhir-naico we don't support geocouch and are not affiliated with couchbase. You could try filing an issue in the geocouch repo.

**Reply by sudhir-naico** [reply]
Thanks for your reply. I asked here to know if the latest CouchDB versions support geoCouch or not. Because the steps mentioned in the geocouch documentation says to execute "bootstrap" from CouchDB source. Also, the document mentioning some paths which are not in the CouchDB source folder (like <path-to-couchdb-source>/etc/couchdb/default.d/).
